### DISCUSSION TITLE
show installedapps of all my clients

### DISCUSSION THREAD

**Discussion by q16marvin** [Ylianst/MeshCentral#6508] [Q&A]
Hi,

new day, new problem :) today I have the challenge to check, on which client is openvpn installed and in which version. That was the reason to find the console command "installedapps", realy awesome!

![grafik](https://github.com/user-attachments/assets/df9576f3-9474-4e01-bb47-d6b014bd68b9)

can i run now this command over "all my clients" to got a list which client uses which openvpn version?

thx for this realy helpful tool :)

**Comment by si458** [comment]
The only way to do something like that would be to use meshctrl.js and automate it

You can use the batch option to run the command on all the machines, but the output gets displays on the console tab of each device so u would have to manually look at it I'm afraid

There is already an open request for a tab to use this command, to display what is installed, etc, but it's a big job because we would need to add extra permissions, etc, for that tab to restrict users

**Reply by q16marvin** [reply]
thank you for your quick response :) i will try to automate it with meshctrl, never used it befor... i will post soltion here, if i find it anyway :)
